cpp/src/cider/exec/operator/join/CiderJoinHashTable.cpp

```diff
@@ -29,13 +29,19 @@ JoinHashTable::JoinHashTable(cider_hashtable::HashTableType hashTableType) {
     case cider_hashtable::HashTableType::LINEAR_PROBING:
       LPHashTableInstance_ =
           std::make_shared<cider_hashtable::LinearProbeHashTable<LP_TEMPLATE>>();
+      break;
     case cider_hashtable::HashTableType::CHAINED:
       chainedHashTableInstance_ =
           std::make_shared<cider_hashtable::ChainedHashTable<CHAINED_TEMPLATE>>();
+      break;
     case cider_hashtable::HashTableType::FIXED_INT8:
       uInt8HashTableInstance_ = std::make_shared<JoinUInt8HashTable>();
+      break;
     case cider_hashtable::HashTableType::FIXED_INT16:
       uInt16HashTableInstance_ = std::make_shared<JoinUInt16HashTable>();
+      break;
+    default:
+      break;
```

Comment:
what happen if no table type matched?

Reply:
It won't happen since `HashTableType` is an enum type. See declaration [HashTableType](https://github.com/Deegue/BDTK/blob/0ad0de0460a20d625219712de774e285d78bb69c/cpp/src/cider/exec/operator/join/HashTableSelector.h#L33).